packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -209,6 +209,14 @@ export default async (cmd: Command) => {
             'Plugin IDs must be lowercase and contain only letters, digits, and dashes.',
           );
         }
+        if (cmd.backend && value.match(/-backend$/)) {
```

Comment:
Tbh I feel the solution below of just silently ignoring the -backend suffix is enough, and we could skip this warning

Reply:
Fair enough. It raises another question though: does it make sense for a non-backend plugin to end with `-backend`? If not then providing either `-backend` suffix or `--backend` flag should yield the same result, WDYT?

Reply:
As a user, if a frontend plugin ended with `-backend`, I would be throughly confused. I'd suggest we avoid allowing that. :)